CustomerData/CartTagging.php

```diff
@@ -41,11 +41,14 @@
 use Magento\Framework\Stdlib\CookieManagerInterface;
 use Magento\Framework\Stdlib\DateTime\DateTime;
 use Nosto\Object\Cart\LineItem;
+use Nosto\Tagging\Helper\Data;
 use Nosto\Tagging\Helper\Scope as NostoHelperScope;
+use Nosto\Tagging\Helper\Scope;
 use Nosto\Tagging\Model\Cart\Builder as NostoCartBuilder;
 use Nosto\Tagging\Model\Customer;
 use Nosto\Tagging\Model\Customer as NostoCustomer;
 use Nosto\Tagging\Model\CustomerFactory as NostoCustomerFactory;
+use Nosto\Tagging\Helper\Url;
```

Comment:
Correct the alias. I think we import it as `NostoHelperUrl` or something like that.